### ISSUE TITLE
Something wrong with log output in CIDER

### ISSUE THREAD

**Issue by agzam** [boot-clj/boot-cljs#138]
### to reproduce:

`cider-jack-in` in any project that uses boot-cljs, with verbose logging option on. Then try to run the task: `boot.user> (boot (dev))`
#### example of a task:

```
(task-options!
  cljs {:ids              #{"js/dev"}
    :compiler-options {:asset-path     "js/dev.out"
               :verbose        true}})

(deftask dev []
  (comp
(serve)
(watch)
(cljs-repl)
(reload :on-jsload 'sample-routing.core/init!)
(speak)
(cljs)
(sift :move {#"dev.js" "main.js"})
(target)))
```
### observed behavior:

boot-cljs's output appears in `*nrepl-server*` buffer and not in the same cider-repl buffer.
### expected behavior:

it should appear in cider-repl buffer. For comparison: lein project with cljs-build outputs log in the same buffer
### env:

CIDER 0.14.0snapshot (package: 20161013.518)

GNU Emacs 25.1.1 (x86_64-apple-darwin15.6.0, NS appkit-1404.47 Version 10.11.6 (Build 15G1004)) of 2016-09-19

BOOT_CLOJURE_VERSION=1.8.0
BOOT_VERSION=2.6.0

[adzerk/boot-cljs              "1.7.228-1"]

**Comment by bhagany** [comment]
This is fixed here: https://github.com/clojure-emacs/cider-nrepl/pull/387

**Comment by Deraen** [comment]
I guess the fix will be included in Cider-nrepl 0.15?

**Comment by bhagany** [comment]
Yes, that's correct.
